emcc.py

```diff
@@ -3021,6 +3025,18 @@ def check_bad_eq(arg):
     elif newargs[i] == '-fno-rtti':
       shared.Settings.USE_RTTI = 0
 
+    # TODO Currently -fexceptions only means Emscripten EH. Switch to wasm
+    # exception handling by default when -fexceptions is given when wasm
+    # exception handling becomes stable.
+    if wasm_eh_enabled:
+      shared.Settings.EXCEPTION_HANDLING = 1
+      shared.Settings.DISABLE_EXCEPTION_THROWING = 1
+      shared.Settings.DISABLE_EXCEPTION_CATCHING = 1
+    elif eh_enabled:
+      shared.Settings.EXCEPTION_HANDLING = 0
+      shared.Settings.DISABLE_EXCEPTION_THROWING = 0
+      shared.Settings.DISABLE_EXCEPTION_CATCHING = 0
```

Comment:
and then we can remove these 8 lines, since the settings would have already been set.

Reply:
I think the reply above contains the answer for this thing too. Please let me know if I didn't understand your comment correctly.